_includes/api/en/4x/routing-args.html

```diff
@@ -36,7 +36,8 @@ <h4> Arguments</h4>
 <p>
 You can provide multiple callback functions that behave just like middleware, except
 that these callbacks can invoke <code>next('route')</code> to bypass
-the remaining route callback(s). You can use this mechanism to impose pre-conditions
+the remaining route callback(s) or invoke <code>next('router')</code> to jump out of the
+current router. You can use this mechanism to impose pre-conditions
```

Comment:
This next sentence needs to be updated now that `'router'` is included in the previous sentence, as it goes on only to explain `'route'` but not `'router'`.